### ISSUE TITLE
matplotlib: missing "plt.get_cmap" and "colors.rgb2hex"

### ISSUE THREAD

**Issue by jli** [microsoft/pylance-release#1037]
<h3> Environment data </h3>

-   Language Server version: 2021.3.1
-   OS and version: darwin x64
-   Python version (and distribution if applicable, e.g. Anaconda): 3.9.2 (and matplotlib==3.3.4)
-   python.analysis.indexing: undefined
-   python.analysis.typeCheckingMode: basic

<h3> Expected behaviour </h3>

The functions`matplotlib.pyplot.get_cmap` and `matplotlib.colors.rgb2hex` should be recognized.

<h3> Actual behaviour </h3>

I get red squiggles.

<h3> Code Snippet / Additional information </h3>

```python
#%%
from matplotlib import cm, colors
from matplotlib import pyplot as plt

def cmap_hex_squiggs(colormap: str, val: float) -> str:
    # "get_cmap" is not a known member of module
    cmap = plt.get_cmap(colormap)
    # "rgb2hex" is not a known member of module
    hex = colors.rgb2hex(cmap(val))
    return hex

def cmap_hex_no_squiggs(colormap: str, val: float) -> str:
    cmap = cm.get_cmap(colormap)
    hex = colors.to_hex(cmap(val))
    return hex
```

Upon googling, it seems like `colors.rgb2hex` was renamed to `to_hex` between matplotlib [1.5.0](https://matplotlib.org/1.5.0/api/colors_api.html) and [2.0.0](https://matplotlib.org/2.0.0/api/colors_api.html), and it seems like `cm.get_cmap` is more canonical (per https://matplotlib.org/stable/tutorials/colors/colormaps.html). But FWIW, these old/alternative names still work.

![image](https://user-images.githubusercontent.com/133466/110936394-d31b0700-82fe-11eb-96c6-bd37185b4121.png)

**Comment by jakebailey** [comment]
Thanks for the report. Note that the stubs we bundle are not yet high quality enough to do strict type checking and are mainly intended to improve the completion experience (versus actually check valid code).

**Comment by jakebailey** [comment]
I've added the members to `colors`; they weren't documented but are listed in the code as there for backwards compatibility.

`get_cmap` is also not documented as in `pyplot` and is only exported due to an import side effect, but it does appear to be there in the real code without any uses, so I'll add it too.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.3.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202132-17-march-2021
